Sources/WalletConnectSign/Engine/Common/SessionEngine.swift

```diff
@@ -3,54 +3,39 @@ import Combine
 import WalletConnectUtils
 import WalletConnectKMS
 
-
 final class SessionEngine {
-    var onSessionRequest: ((Request)->())?
-    var onSessionResponse: ((Response)->())?
-    var onSessionSettle: ((Session)->())?
-    var onSessionRejected: ((String, SessionType.Reason)->())?
-    var onSessionDelete: ((String, SessionType.Reason)->())?
-    var onEventReceived: ((String, Session.Event, Blockchain?)->())?
-    
-    var settlingProposal: SessionProposal?
     
+    typealias SessionRequestCallback = (Request) -> Void
+    typealias SessionResponseCallback = (Response) -> Void
+    typealias SessionRejectedCallback = (String, SessionType.Reason) -> Void
+    typealias SessionDeleteCallback = (String, SessionType.Reason) -> Void
+    typealias EventReceivedCallback = (String, Session.Event, Blockchain?) -> Void
```

Comment:
I'm not a fan of spreading too many typealiases through the code. They can make it harder to understand the code when used excessively.